Source/bmalloc/libpas/src/libpas/pas_lock.h

```diff
@@ -179,7 +179,12 @@ static inline void pas_lock_lock(pas_lock* lock)
     pas_race_test_will_lock(lock);
 #if PAS_USE_ULOCK_SPI
     const os_unfair_lock_options_t options = (os_unfair_lock_options_t)(OS_UNFAIR_LOCK_DATA_SYNCHRONIZATION | OS_UNFAIR_LOCK_ADAPTIVE_SPIN);
+#if defined(OS_UNFAIR_LOCK_INLINE) && OS_UNFAIR_LOCK_INLINE
     os_unfair_lock_lock_with_options_inline(&lock->lock, options);
+#else
+    PAS_UNUSED_PARAM(options);
+    PAS_ASSERT_NOT_REACHED();
```

Comment:
When I originally put together the bmalloc modulemap, I assumed (perhaps incorrectly) that we needed to maintain the property of a _compile-time_ failure if `OS_UNFAIR_LOCK_INLINE` was not defined. I'm glad if that's no longer the case...